.circleci/config.yml

```diff
@@ -90,7 +90,7 @@ jobs:
           name: get remark styles
           command: |
            cd ~
-           cat ~/build/npm-requirements.txt | xargs npm install
+           cat ~/project/npm-requirements.txt | xargs npm install
```

Comment:
It doesn't look like there will be anything in `~/project`? This doesn't make much sense to me.

Reply:
I just checked running the ssh on 1326 and there is a project folder. This folder is created at [checkout](https://github.com/franklin-feingold/bids-specification/blob/circle_npm/.circleci/config.yml#L76)

Reply:
`~/project` is the default CircleCI working directory, search for it (ctrl+f) on this page: https://circleci.com/docs/2.0/configuration-reference/ 

if you tested this and it works, this seems fine to me